wagtail/admin/templates/wagtailadmin/pages/side_panels/preview.html

```diff
@@ -23,6 +23,18 @@
     <button class="button button-small button-secondary button--icon preview-panel__refresh-button" data-refresh-preview>{% icon name="rotate" %} Refresh</button>
 
     <div class="preview-panel__area" data-preview-panel-area>
+        <div class="preview-panel__error-banner">
+            <p class="preview-panel__error-title">
+                {% blocktrans %}
```

Comment:
```suggestion
                {% blocktrans trimmed %}
```
everywhere please.
Also, if not using any variables in the translated text, `{% trans "This preview is not up-to-date due to validation errors." %}` may be better

Reply:
Thanks! This was just a temporary implementation as the design is not available yet, but it's good to keep this in mind. I'll check if there are other instances of this.